mappings/net/minecraft/data/server/RecipesProvider.mapping

```diff
@@ -21,84 +23,143 @@ CLASS net/minecraft/class_2446 net/minecraft/data/server/RecipesProvider
+		ARG 0 exporter
 		ARG 1 outputItem
 		ARG 2 inputItem
 	METHOD method_24894 offerConcretePowderDyeingRecipe (Ljava/util/function/Consumer;Lnet/minecraft/class_1935;Lnet/minecraft/class_1935;)V
+		ARG 0 exporter
 		ARG 1 outputItem
 		ARG 2 inputItem
 	METHOD method_29728 offerNetheriteUpgradeRecipe (Ljava/util/function/Consumer;Lnet/minecraft/class_1792;Lnet/minecraft/class_1792;)V
+		ARG 0 exporter
 		ARG 1 outputItem
 		ARG 2 inputItem
 	METHOD method_32233 offerCandleDyeingRecipe (Ljava/util/function/Consumer;Lnet/minecraft/class_1935;Lnet/minecraft/class_1935;)V
+		ARG 0 exporter
 		ARG 1 outputItem
 		ARG 2 inputItem
 	METHOD method_32804 createSlabRecipe (Lnet/minecraft/class_1935;Lnet/minecraft/class_1856;)Lnet/minecraft/class_5797;
 		ARG 0 outputItem
+		ARG 1 inputItem
 	METHOD method_32805 createChiseledBlockRecipe (Lnet/minecraft/class_1935;Lnet/minecraft/class_1856;)Lnet/minecraft/class_2447;
 		ARG 0 outputItem
+		ARG 1 inputItem
 	METHOD method_32806 createPressurePlateRecipe (Lnet/minecraft/class_1935;Lnet/minecraft/class_1856;)Lnet/minecraft/class_5797;
 		ARG 0 outputItem
+		ARG 1 inputItem
 	METHOD method_32807 hasItem (Lnet/minecraft/class_1935;)Ljava/lang/String;
 		ARG 0 item
 	METHOD method_32808 createStairsRecipe (Lnet/minecraft/class_1935;Lnet/minecraft/class_1856;)Lnet/minecraft/class_5797;
 		ARG 0 outputItem
 		ARG 1 inputItem
 	METHOD method_32809 offerWallRecipe (Ljava/util/function/Consumer;Lnet/minecraft/class_1935;Lnet/minecraft/class_1935;)V
+		ARG 0 exporter
 		ARG 1 outputItem
 		ARG 2 inputItem
 	METHOD method_32810 offerPolishedStoneRecipe (Ljava/util/function/Consumer;Lnet/minecraft/class_1935;Lnet/minecraft/class_1935;)V
+		ARG 0 exporter
 		ARG 1 outputItem
 		ARG 2 inputItem
 	METHOD method_32811 offerCutCopperRecipe (Ljava/util/function/Consumer;Lnet/minecraft/class_1935;Lnet/minecraft/class_1935;)V
+		ARG 0 exporter
 		ARG 1 outputItem
 		ARG 2 inputItem
 	METHOD method_32812 offerChiseledBlockRecipe (Ljava/util/function/Consumer;Lnet/minecraft/class_1935;Lnet/minecraft/class_1935;)V
+		ARG 0 exporter
 		ARG 1 outputItem
 		ARG 2 inputItem
 	METHOD method_32813 createPressurePlateRecipe (Ljava/util/function/Consumer;Lnet/minecraft/class_1935;Lnet/minecraft/class_1935;)V
+		ARG 0 exporter
 		ARG 1 outputItem
 		ARG 2 inputItem
 	METHOD method_32814 offerSlabRecipe (Ljava/util/function/Consumer;Lnet/minecraft/class_1935;Lnet/minecraft/class_1935;)V
+		ARG 0 exporter
 		ARG 1 outputItem
 		ARG 2 inputItem
+	METHOD method_33531 getWallRecipe (Lnet/minecraft/class_1935;Lnet/minecraft/class_1856;)Lnet/minecraft/class_5797;
+		ARG 0 outputItem
+		ARG 1 inputItem
+	METHOD method_33533 getVariantRecipeInput (Lnet/minecraft/class_5794;Lnet/minecraft/class_5794$class_5796;)Lnet/minecraft/class_2248;
+		COMMENT Gets the block used to craft a certain {@linkplain variant BlockFamily.Variant} of a base block.
```

Comment:
Shouldn't it be
```suggestion
		COMMENT Gets the block used to craft a certain {@linkplain BlockFamily.Variant variant} of a base block.
```

Reply:
Fixed in c9c561c58d9da83f0cbc0e4c47a973fd570e3d7c